lib/spack/external/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 * Homepage: https://pypi.python.org/pypi/archspec
 * Usage: Labeling, comparison and detection of microarchitectures
-* Version: 0.1.2 (commit 147942bd96219cf39e7001aef3644b43af76bdf6)
```

Comment:
It's very likely we'll be releasing archspec v0.1.4 with this commit in the next few days, so I already updated the version number here.